### ISSUE TITLE
Regarding including minimal implementations of research papers using TensorFlow 2 (and Keras)

### ISSUE THREAD

**Issue by sayakpaul** [tensorflow/models#8589]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://github.com/tensorflow/models/tree/master/research/...

## 2. Describe the issue

Hi.

I am interested to know if it is possible to enlist the minimal implementations of the research papers using TensorFlow 2 and most importantly `tf.keras`. I have done this for a number of different research papers including the following:
- SimCLR ([paper](https://arxiv.org/abs/2002.05709) | [my minimal implementation](https://github.com/sayakpaul/SimCLR-in-TensorFlow-2))
- Training BatchNorm and only BatchNorm ([paper](https://arxiv.org/abs/2003.00152) | [my minimal implementation](https://github.com/sayakpaul/Training-BatchNorm-and-Only-BatchNorm/))
- EvoNorm Layers ([paper](https://arxiv.org/abs/2004.02967) | [my minimal implementation](https://github.com/sayakpaul/EvoNorms-in-TensorFlow-2))
- NALU ([paper](https://arxiv.org/abs/1808.00508) | [my minimal implementation](https://github.com/sayakpaul/TF-2.0-Hacks/tree/master/NALU%20in%20TF%202.0))
- Supervised Contrastive Learning ([paper](https://arxiv.org/abs/2004.11362) | [my minimal implementation](https://github.com/sayakpaul/Supervised-Constrastive-Learning-in-TensorFlow-2))*

I think this might be helpful for a number of different people and folks looking to use TensorFlow 2 and `tf.keras` in systematic ways. Looking forward to hearing your thoughts on this. 

*This is a joint effort between myself and Shweta Shaw.

**Comment by jaeyounkim** [comment]
@sayakpaul I will reach out to you by email to discuss details on your implementations.

**Comment by bhack** [comment]
What was the outcome of this? Can you share something about this thread?
We have also SimCLRv2 now in https://github.com/google-research/simclr

**Comment by sayakpaul** [comment]
@jaeyounkim

**Comment by jaeyounkim** [comment]
We are not accepting minimal implementations for the research directory.

**Comment by sayakpaul** [comment]
Well, I **_never asked_** to accept the minimal implementations for the research directory. I asked if it's possible to give mentions because people might find minimal implementations (done in Colab Notebooks) easier to follow along and grasp. It turned out it's not possible at this point in time.
